packages/schema/src/parser.ts

```diff
@@ -20,11 +20,11 @@ export class AsnParser {
    */
   public static parse<T>(data: BufferSource, target: IEmptyConstructor<T>): T {
     let buf: ArrayBuffer;
-    if (data instanceof ArrayBuffer) {
+    if (isArrayBuffer(data)) {
       buf = data;
     } else if (typeof Buffer !== "undefined" && Buffer.isBuffer(data)) {
       buf = new Uint8Array(data).buffer;
-    } else if (ArrayBuffer.isView(data) || (data as any).buffer instanceof ArrayBuffer) {
+    } else if (ArrayBuffer.isView(data) || isArrayBuffer((data as any).buffer)) {
```

Comment:
I didn't need this change to make my packages work, but used the new `isArrayBuffer()` function for consistency only.